README.md

```diff
@@ -2,6 +2,7 @@
 
 - [Visual Studio Professional 2019](https://visualstudio.microsoft.com/downloads/)
     - Make sure you have the [NPM Task Runner](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.NPMTaskRunner) extension
+    - JetBrains Rider can work as an alternative. Follow the setup steps laid out below. In addition, you'll have to run `npm run dev` manually to build the JS and SASS.

```

Comment:
Do you think it would be useful to:
- Add a link to Rider (like we have for VS)
- Specify that it's an alternative to VS rather than alternative to NPM Task Runner to be extra clear.

Reply:
Good points - done.